ci_framework/roles/openshift_provisioner_node/tasks/add_bridges.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create the external network connection
+  tags:
+    - bootstrap
+  become: true
+  community.general.nmcli:
+    type: bridge
+    conn_name: "{{ cifmw_opn_external_bridge_name }}"
+    stp: false
+    state: present
+
+- name: Verify external network interface is defined
+  tags:
+    - bootstrap
+  when:
+    - cifmw_opn_external_network_iface is undefined
+    - not cifmw_opn_dry_run | bool
+  ansible.builtin.fail:
+    msg: "cifmw_opn_external_network_iface is not defined"
+
+- name: Add iface to the external network connection
+  tags:
+    - bootstrap
+  when: not cifmw_opn_dry_run | bool
+  become: true
+  ansible.builtin.script:
+    cmd: files/add_bridge_port.sh "{{ cifmw_opn_external_bridge_name }}" "{{ cifmw_opn_external_network_iface }}"
+
+- name: Provisioning network configuration
+  tags:
+    - bootstrap
+  when:
+    - cifmw_opn_use_provisioning_network is defined
+    - cifmw_opn_use_provisioning_network | bool
+    - cifmw_opn_prov_network_iface is defined
+  block:
+    - name: Create the provisioning network connection
```

Comment:
Dumb suggestion. Can these two tasks being extracted to a yml file, maybe called `add_single_bridge` or something similar and reuse them? Is not a must comment, just an idea, as this is already a reusable task file. I see that the only difference between the two bridges is the `ip4` parameters, that may be optional by using omit.

Reply:
Initially, I did try but faced an error when `ipv4` was empty. As it caused the first bridge to be set to `ipv4.method = manual` which prevented DHCP configurations hence the split.

Reply:
Is it okay, if I pick up this suggestion in another PR (i.e. using `omit`)?

Reply:
+1 for follow-up